README.md

```diff
@@ -36,3 +36,12 @@ the rewritten target URL might look like this:
 
 and the text 'About the REDCap Mobile App' will be added on the 'REDCap Mobile App' before the title 'What is the REDCap Mobile App?'.
 
+
+## Module Workflow
+1. The user requests a token: On the project in question, the user clicks on the left menu link **REDCap Mobile App** and then on the button **Request API token**. He is redirected to the URL provided during the configuration of the module. Once he completes the questionnaire, the token request will be submitted.
+2. The project manager/administrator receives the request.
+3. The project manager/administrator approves the request.
+4. The project manager/administrator disables the module: The project manager/administrator disables the module in the project by going to the ** External Modules** page (under Applications) and clicking **Disable** next to the **Route Mobile Api Token Requests to Survey** module.
+5. The user goes back to the project in question and clicks the **Request API token** button on the **REDCap Mobile App** page.
```

Comment:
I think step 5 needs to be preceded by a new step that reads something like this

```
The project manager/administrator contacts the user: The project manager/administrator sends a message to the user asking him to revisit the mobile API token request page to complete the token request.
```